addons/collapse-footer/addon.json

```
@@ -0,0 +1,40 @@
+{
+  "name": "Sticky footer",
+  "description": "Makes the website footer stick out slightly from the bottom of the window and makes it expand when hovering over it. Previously part of \"Infinite scolling\".",
```

Potential description extension:
```suggestion
  "description": "Makes the website footer stick out slightly from the bottom of the window and makes it expand when hovering over it, allowing it to be quickly accessed from anywhere on the page. Previously part of \"Infinite scolling\".",
```